Eclipse/semi-compilado/src/net/javaguides/usermanagement/model/Leito.java

```diff
@@ -30,18 +37,11 @@ public Leito(int id, String medico, String paciente, String enfermeiro) {
 		this.ocupado = true;
 	}
 
-	/*public Leito(int id, String name, String email, String country) {
-		super();
-		this.id = id;
-		this.name = name;
-		this.email = email;
-		this.country = country;
-	}*/
-
 	public int getId() { return this.id; }
 	public String getMedico() { return this.medico; }
 	public String getPaciente() { return this.paciente; }
 	public String getEnfermeiro() { return this.enfermeiro; }
+	public String getHospital() { return this.hospital; }
```

Comment:
é bom add um set tbm
```suggestion
	public String getHospital() { return this.hospital; }
	public void setHospital(String hospital){  this.hospital = hospital; }
```